### PR TITLE
forge2d workshop short link

### PR DESCRIPTION
**Pull request by khanhnwin**
Creating a new short link for Brett's forge2d workshop

### CODE REVIEW

File: firebase.json [hunk 1]
```diff
@@ -389,6 +389,7 @@
       { "source": "/go/flutter-tool-extension-api", "destination": "https://docs.google.com/document/d/12iXm23TP6e6qS1Ntx-i-azcBWByiUl_kWjRxRC2YTz8/edit?usp=sharing&resourcekey=0-pVl2giJzGAHqOYEaZagVow", "type": 301 },
       { "source": "/go/flutter-web-scenelets", "destination": "https://docs.google.com/document/d/1GUAx3aqdtEoaBMTNpsS1-i59QZlwTi6vUCASbaanPCo", "type": 301 },
       { "source": "/go/focus-management", "destination": "https://docs.google.com/document/d/1-NS3NJFd2j9kzKj5GOPoWq-G0B6jrpiCL892EX5Nxgk/edit", "type": 301 },
+      { "source": "/go/forge2d-workshop", "destination": "https://codelabs.developers.google.com/codelabs/flutter-flame-forge2d", "type": 301 },
```
Comment (domesticmouse) [inline firebase.json:392]: Can we make the destination https://codelabs.developers.google.com/codelabs/flutter-flame-forge2d#0 to force loading of the introduction page?

**Review by domesticmouse — APPROVED** [review]

**Review by sfshaza2 — APPROVED** [review]
lgtm, once the first page of the codelab is forced.

File: firebase.json (hunk 1 above)
Comment (khanhnwin) [reply firebase.json]: done!

**Review by domesticmouse — APPROVED** [review]
WOOHOO 

Thanks @khanhnwin and @sfshaza2 =)

**Pull request merged**